src/components/TableHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { SortLink } from './SortLink';
+
+type Props = {
+  columnName: string;
+  sortColumn: string;
+  isReversed: boolean;
+  sortParam: string | null;
+};
+
+export const TableHeader: React.FC<Props> = ({
+  columnName,
+  sortColumn,
+  isReversed,
+  sortParam,
+}) => (
+  <span className="is-flex is-flex-wrap-nowrap">
+    {columnName}
```

Comment:
Why it's called `TableHeader` but I see only a single span here and no related to table tags? 👀

Reply:
> Why it's called `TableHeader` but I see only a single span here and no related to table tags? 👀

Not fixed